<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>151</width>
      <height>361</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>照片：</string>
    </property>
    <widget class="QLabel" name="dbLabPhote">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>535</width>
      <height>366</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QGroupBox" name="groupBoxSort">
           <property name="title">
            <string>排序</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>排序字段:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboFields"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QRadioButton" name="radioButton">
                  <property name="text">
                   <string>升序</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="radioButton_2">
                  <property name="text">
                   <string>降序</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBoxFilter">
           <property name="title">
            <string>数据过滤</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QRadioButton" name="radioButton_3">
                  <property name="text">
                   <string>男</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="radioButton_4">
                  <property name="text">
                   <string>女</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_5">
                <property name="text">
                 <string>全显示</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QTableView" name="tableView"/>
      </widget>
     </item>
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>工号：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>姓名：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>性别：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>身高：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>出生日期：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>手机号：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>出生省份：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>城市：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>部门：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>学历：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>工资：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>备注：</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QSpinBox" name="dbSpinEmpNo"/>
         </item>
         <item>
          <widget class="QLineEdit" name="dbEditName"/>
         </item>
         <item>
          <widget class="QComboBox" name="dbComboSex"/>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="dbSpinHeight"/>
         </item>
         <item>
          <widget class="QDateEdit" name="dbEditBirth"/>
         </item>
         <item>
          <widget class="QLineEdit" name="dbEditMobile"/>
         </item>
         <item>
          <widget class="QComboBox" name="dbComboProvince"/>
         </item>
         <item>
          <widget class="QLineEdit" name="dbEditCity"/>
         </item>
         <item>
          <widget class="QComboBox" name="dbComboDep"/>
         </item>
         <item>
          <widget class="QComboBox" name="dbComboEdu"/>
         </item>
         <item>
          <widget class="QSpinBox" name="dbSpinSalary"/>
         </item>
         <item>
          <widget class="QLabel" name="dbEditMemo">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>734</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actOpenDB"/>
   <addaction name="actRecAppend"/>
   <addaction name="actRecInsert"/>
   <addaction name="actRecDelete"/>
   <addaction name="actPhoto"/>
   <addaction name="actPhotoClear"/>
   <addaction name="actScan"/>
   <addaction name="actSubmit"/>
   <addaction name="actRevert"/>
   <addaction name="actQuit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actOpenDB">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actRecAppend">
   <property name="text">
    <string>添加</string>
   </property>
  </action>
  <action name="actRecInsert">
   <property name="text">
    <string>插入</string>
   </property>
  </action>
  <action name="actSubmit">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actRevert">
   <property name="text">
    <string>取消</string>
   </property>
  </action>
  <action name="actRecDelete">
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="actPhoto">
   <property name="text">
    <string>设置照片</string>
   </property>
  </action>
  <action name="actPhotoClear">
   <property name="text">
    <string>清除照片</string>
   </property>
  </action>
  <action name="actScan">
   <property name="text">
    <string>涨工资</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
